content/hi/container-image.md

```diff
@@ -0,0 +1,11 @@
+---
+title: कंटेनर इमेज
+status: Feedback Appreciated
+category: अवधारणा
+---
+
+एक कंटेनर इमेज (छवि) एक अपरिवर्तनीय, स्थिर फ़ाइल है जिसमें एक कंटेनर के निर्माण के लिए निर्भरताएँ होती हैं। इन निर्भरताओं में एकल निष्पादन योग्य बाइनरी फ़ाइल, सिस्टम लाइब्रेरी, सिस्टम उपकरण, एनवायरनमेंट वेरिएबल्स और अन्य आवश्यक प्लेटफ़ॉर्म समायोजन शामिल हो सकते हैं। कंटेनर इमेजिस एक एप्लिकेशन के कंटेनरीकरण से उत्पन्न होती हैं और आमतौर पर कंटेनर रजिस्ट्रियों में संग्रहीत की जाती हैं, जहां उन्हें कंटेनर रनटाइम इंटरफ़ेस (CRI) का उपयोग करके एक अलग प्रक्रिया के रूप में डाउनलोड और चलाया जा सकता है। एक कंटेनर इमेज ढांचे को ओपन कंटेनर इनिशिएटिव (OCI) द्वारा परिभाषित मानक स्कीमा का पालन करना चाहिए।
```

Comment:
```suggestion
एक कंटेनर इमेज (छवि) एक अपरिवर्तनीय, स्टैटिक फ़ाइल है जिसमें एक कंटेनर के निर्माण के लिए निर्भरताएँ होती हैं। इन निर्भरताओं में एक एक्सेक्युट करने योग्य बाइनरी फ़ाइल, सिस्टम लाइब्रेरी, सिस्टम उपकरण, एनवायरनमेंट वेरिएबल्स और अन्य आवश्यक प्लेटफ़ॉर्म समायोजन शामिल हो सकते हैं। कंटेनर इमेज एक एप्लिकेशन के कंटेनरीकरण से उत्पन्न होती हैं और आमतौर पर कंटेनर रजिस्ट्रियों में संग्रहीत की जाती हैं, जहां उन्हें कंटेनर रनटाइम इंटरफ़ेस (CRI) का उपयोग करके एक अलग प्रक्रिया के रूप में डाउनलोड और चलाया जा सकता है। एक कंटेनर इमेज ढांचे को ओपन कंटेनर इनिशिएटिव (OCI) द्वारा परिभाषित मानक स्कीमा का पालन करना चाहिए।
```

Reply:
I think we should not translate "static" and "execute" as they being technical terms are always used in their English version and may get difficult to understand what we are saying upon translation

Reply:
Completely agree! Will also add these to the spreadsheet. Thank you !